gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc

```diff
@@ -304,77 +303,119 @@ You should see a list of services. Flex Gateway is successfully running if each
 ----
 
 // end::linux-start-commands[]
-// tag::k8s-install-helm-chart[]
+// tag::k8s-install-flex-helm-chart-title[]
 
 === Install Helm Chart into the Namespace
 
-Before installing, ensure that you have:
-
-* https://helm.sh/docs/intro/install/[Helm^], a tool used to install Flex Gateway, monitoring tools, and applications. A minimum Helm version of 3.0.0 is required. 
-+
-Refer to the <<helm-chart-options,Helm Chart Configuration Options>> for information about customizing the chart.
-
-. Add the Flex Gateway Helm repository:
+// end::k8s-install-flex-helm-chart-title[]
+// tag::k8s-install-flex-helm-chart-intro-connected[]
+Use Helm to deploy Flex Gateway to a node in your cluster and connect it to Anypoint Platform. After deploying, use Runtime Manager to verify that the gateway is present and connected to Anypoint Platform.
+// end::k8s-install-flex-helm-chart-intro-connected[]
+// tag::k8s-install-flex-helm-chart-intro-local[]
+Use Helm to deploy Flex Gateway to a node in your cluster and connect it to Anypoint Platform. After deploying, use Runtime Manager to verify that the gateway is present and connected to Anypoint Platform.
+// end::k8s-install-flex-helm-chart-intro-local[]
+// tag::k8s-flex-helm-chart[]
+A Helm chart installs Flex Gateway, monitoring tools, and applications.
+// end::k8s-flex-helm-chart[]
+// tag::k8s-flex-helm-chart-defaults[]
+Refer to <<helm-chart-options>> for configuration options. 
+// end::k8s-flex-helm-chart-defaults[]
+
+. Add a Helm repository named `flex-gateway` for your chart: 
 +
-[source,kubernetes]
+[source,helm]
 ----
 helm repo add flex-gateway https://flex-packages.anypoint.mulesoft.com/helm
 ----
++
+The command either adds the repository or skips this process if a Helm repository with that name already exists on your machine:
++
+* If the repository is new, the command returns the following message:
++
+----
+"flex-gateway" has been added to your repositories
+----
++
+* If the repository already exists, the command returns the following message:
++
+----
+"flex-gateway" already exists with the same configuration, skipping
+----
 
-. Update the Helm repository using the following command:
+. Run `helm repo up`.
++
+The command returns the following message:
 +
-[source,kubernetes]
 ----
-helm repo up
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "flex-gateway" chart repository
+Update Complete. ⎈Happy Helming!⎈
 ----
++
+If you have more than one Helm repository on your machine, the message in your terminal window lists all of the repositories.
+// end::k8s-flex-helm-chart-add[]
+// tag::k8s-flex-helm-chart-deploy[]
+//      PLEASE retain blank line before first step below
 
-. Using Ingress, install the *flex-gateway* Helm chart into the *gateway* namespace.
+. Run the Helm command for deploying your gateway and connecting it to Anypoint Platform:
```

Comment:
TBD: Determine general wording or split, as "and connecting it to Anypoint Platform" will be in local mode pages, as well as connected.